lib/davinci_dtr_test_kit/profiles/questionnaire_response/questionnaire_response_create.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../create_test'
+
+module DaVinciDTRTestKit
+  class QuestionnaireResponseCreateTest < Inferno::Test
+    include DaVinciDTRTestKit::CreateTest
+
+    title 'Server is capable of creating a QuestionnaireResponse resource from QuestionnaireResponse create interaction'
+    description 'A sever SHALL support the QuestionnaireResponse create interaction'
+
+    id :questionnaire_response_create
+    input :create_questionnaire_resources
```

Comment:
Flesh out this input a bit more - at minimum is should have the textarea type since it is a big block of content. Applies to the other similar create and update inputs.